src/internal/constants.ts

```diff
@@ -0,0 +1,9 @@
+export const PLAGIN_NAME = "@solarity/hardhat-zkit";
+
+export const CIRCOM_CIRCUITS_CACHE_FILENAME = "circom-circuits-cache.json";
+
+export const INCLUDE_REG_EXP = /include +"([.\-/\w]+)";/g;
```

Comment:
Can it be single quoted?

Reply:
No, only double quotes are supported by the compiler